sdk/core/azure-core/src/main/java/com/azure/core/util/polling/DefaultSyncPoller.java

```diff
@@ -37,42 +37,38 @@
     /**
      * Creates DefaultSyncPoller.
      *
-     * @param defaultPollInterval the default polling interval
-     * @param activationOperation the activation operation to be invoked at most once across all subscriptions,
-     *                            this parameter is required, if there is no specific activation work to be
-     *                            done then invocation should return Mono.empty(), this operation will be called
-     *                            with a new {@link PollingContext}.
+     * @param pollInterval the polling interval
+     * @param syncActivationOperation the operation to synchronously activate (start) the long running operation,
+     *     this operation will be called with a new {@link PollingContext}.
```

Comment:
This still lacks clarity to me. I don't know, as a user of this API, what exactly I should supply here. In other words, you're telling me how this is used, rather than how I use it.